src/critic.lisp

```diff
@@ -271,6 +307,38 @@ To learn more about using it as a part of the GitHub workflow, read
    You can ignore all `let*-single` warnings by adding `--ignore 'let*-single'`
    command line option or put a special comment before the top-level form:
 
+   You can ignore all `let*-single` warnings by adding `--ignore 'let*-single'`
+
+   ```bash
+   lisp-critic --ignore 'let*-single' lisp-critic
+   ```
+
+   or ignore all `if-no-else` and `needless-shiftf` warnings by adding
+
+   ```bash
+   lisp-critic --ignore 'if-no-else,needless-shiftf' lisp-critic
+   ```
+
+   in the command line. Alternatively you can use the short version `-i`
+   instead of `--ignore`.
+
+   You can whitelist `let*-single` warnings by adding `--whitelist 'let*-single'`
+
+   ```bash
+   lisp-critic --whitelist 'let*-single' lisp-critic
+   ```
+
+   or whitelist `if-no-else` and `needless-shiftf` warnings by adding
+
+   ```bash
+   lisp-critic --whitelist 'if-no-else,needless-shiftf' lisp-critic
+   ```
+
+   in the command line. Alternatively you can use the short version `-w`
+   instead of `--whitelist`.
+
+   To ignore a top-level-form, you can put a special comment before:
+
```

Comment:
Seems here should go an example.

Reply:
[73493a7](https://github.com/40ants/40ants-critic/pull/1/commits/73493a7f4ec12f8d37f70515813d1e683b3473a6) fixes this